scripts/common/host-packages.sh

```diff
@@ -95,6 +95,22 @@ function dpkg_install_host_packages() {
     _dpkg_install_host_packages "$dir" "$dir_prefix" "${packages[@]}"
 }
 
+function _dpkg_apt_get_status_and_maybe_fix_broken_pkgs() {
+    logStep "Checking package status"
+    if  sudo apt-get check status ; then
```

Comment:
Do we need to use `sudo` here ? I think we are already root.

Reply:
Yes, we need the sudo otherwise we will check Permission denied

Reply:
<img width="965" alt="Screenshot 2022-12-19 at 14 39 59" src="https://user-images.githubusercontent.com/7708031/208450717-dce560ef-df23-45a6-ac02-4c42a5847117.png">

Reply:
That is true |o|